docs/source/developers/guide/communication.rst

```diff
@@ -0,0 +1,77 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. _communication:
+
+*************
+Communication 
+*************
+
+**About the team**
+
+The team is full of experts, software engineers and core developers
+but also full of users, learners and enthusiasts that love doing what we
+do and we want to spread this enthusiasm to you also.
+
+We all have silly questions and we all need help lots of times.
+We encourage you to be open to communication and we will help as much as
+we can.
+
+Where to get help 👋
+====================
```

Comment:
What do you think about also having a sub-section of recommended learning resources? This could be articles on concepts important to Arrow (SIMD, memory management) as well as recommended books for learning languages. For example, I'd be particularly interested to hear what C++ book people recommend that best prepares you for the Arrow codebase.

Reply:
I like this idea!! But would maybe add it to the [Additional info and resources](https://github.com/apache/arrow/pull/11677/files#diff-6c4c5ee85b737e0be4c763213c9da307cab45ae96302e1266b1e1bdc4099225a)?